README.md

```diff
@@ -198,7 +198,13 @@ IPU ES2K target.
    git checkout ipdk_v23.07
    ```
 
-3. Build P4-K8s binaries and container images.
+3. Update `inframanager/config.yaml` file with the arp-mac of the ARP proxy
+   interface's MAC. This should be the interface which user intends to select
+   in the step 3 of the Deploy P4 Kubernetes section. Any other changes to
+   the config.yaml file if needed need to go in this section as well.
```

Comment:
This sounds like update of documentation rather than the steps themselves. Suggest revising the wording e.g. any other changes to config file needs to be done before carrying out next step.

Reply:
Please check now